src/SLCore/Common/Helpers/LanguageExtensions.cs

```diff
@@ -19,26 +19,44 @@
  */
 
 using SonarLint.VisualStudio.SLCore.Common.Models;
+using static SonarLint.VisualStudio.Core.Language;
```

Comment:
Not a fan of static usings. If you need to differentiate between languages you can use `using CoreLanguage = ...;` & `using SLCoreLanguage = ...;`